### PR TITLE
Added policy to check public access key for rds

### PR DESCRIPTION
**Pull request by SanaaYousaf**

### CODE REVIEW

File: avd_docs/kubernetes/general/AVD-KSV-0107/docs.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 
-apiVersion and kind has been deprecated
+apiVersion '' and kind '' has been deprecated on: '' and planned for removal on:''
```
Comment (simar7) [inline avd_docs/kubernetes/general/AVD-KSV-0107/docs.md:2]: Is this change part of the PR?

Comment (SanaaYousaf) [reply avd_docs/kubernetes/general/AVD-KSV-0107/docs.md:2]: Yes, this change is automatically generated by running make docs command

File: avd_docs/aws/apigateway/AVD-AWS-0179/docs.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+### Impact
+Reduce the number of calls made to your API endpoint and also improve the latency of requests to your API with response caching.
+
+<!-- DO NOT CHANGE -->
+{{ remediationActions }}
+
+### Links
+- https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-caching.html
+
+
```
Comment (simar7) [inline avd_docs/aws/apigateway/AVD-AWS-0179/docs.md:13]: why is this part of this PR?

File: avd_docs/aws/apigateway/AVD-AWS-0179/Terraform.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+     logging_level   = "INFO"
+     caching_enabled = true
+   }
+ }
+ 
+```
+
+#### Remediation Links
+ - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_method_settings#cache_enabled
+
```
Comment (simar7) [inline avd_docs/aws/apigateway/AVD-AWS-0179/Terraform.md:29]: ditto

Comment (SanaaYousaf) [reply avd_docs/aws/apigateway/AVD-AWS-0179/Terraform.md]: I deleted these files, But these files are auto generated by running make docs. Am i doing something wrong?

Comment (simar7) [reply avd_docs/aws/apigateway/AVD-AWS-0179/Terraform.md]: Not sure, there's a few rules which seem to have this problem (I'm opening a PR now to fix that) but not these ones. Make sure your feature branches are up to date with the master.

**Review by simar7 — APPROVED** [review]
lgtm thanks!

**Pull request merged**